js_modules/dagit/packages/core/src/assets/AssetPartitions.tsx

```diff
@@ -33,21 +34,29 @@ interface Props {
   opName?: string | null;
 }
 
+const DISPLAYED_STATES = [PartitionState.MISSING, PartitionState.SUCCESS];
+
 export const AssetPartitions: React.FC<Props> = ({
   assetKey,
-  assetPartitionNames,
+  assetPartitionDimensions,
   assetLastMaterializedAt,
   params,
   setParams,
   liveData,
 }) => {
   const [assetHealth] = usePartitionHealthData([assetKey], assetLastMaterializedAt);
-  const [ranges, setRanges] = usePartitionDimensionRanges(assetHealth, assetPartitionNames);
+  const [ranges, setRanges] = usePartitionDimensionRanges(
+    assetHealth,
+    assetPartitionDimensions,
+    true,
```

Comment:
Boolean function params are a mystery -- at a glance I can't tell what this is or is not going to do. :) Perhaps it would help to make the function parameter a config object?

Reply:
Good cal! I think it's time to make this an options object 🙌